### PR TITLE
Add amazon firehose documentation.

### LINKED ISSUE #931: Document AWS Firehose listener
Basically document this: https://github.com/gravwell/gravwell/pull/1020

Please add screenshots on configuration if possible.

### PR DESCRIPTION
**Pull request by david-fritz-gravwell**
Fixes #931

### CODE REVIEW

File: ingesters/http.md [hunk 1]
```diff
@@ -370,6 +374,19 @@ Multiple Tag-Match pairs can be given:
...
+The `Amazon-Firehose-Listener` type supports the [Amazon Firehose](https://aws.amazon.com/firehose/) API. The Firehose API requires the use of HTTPS and an authentication token. 
+
+```
+[Amazon-Firehose-Listener "foo"]
+	URL="/foo"
+	TokenValue="thisisyourtoken" #set the access control token
+	Tag-Name=bar
+```
+
+In the above example, the HTTP ingester will listen on the `/foo` path for an Amazon Firehose request, authenticated with the token "thisisyourtoken", and ingesting to tag "bar". In the AWS console, you would set the Firehose endpoint to `your.domain/foo`, and provide the same token.
```
Comment (kris-watts-gravwell) [inline ingesters/http.md:388]: Please add the config parameter table here too, verbose is good

- URL
- TokenValue
- Tag-Name
- Ignore-Timestamps
- Preprocessor set

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
Add the parameter table, same way we do in HEC

**Comment by david-fritz-gravwell** [comment]
@kris-watts-gravwell PTAL

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**